src/components/GameGrid/GameBoard.tsx

```diff
@@ -32,11 +37,46 @@ const GameBoard: React.FC<Props> = ({ gameData }: Props) => {
       <TextContainer>
         <Shout>Other games</Shout>
 
-        <Loud>Knockout rounds</Loud>
+        <Loud>Round of 32</Loud>
         <Paragraph>Knockout rounds begin on Friday 15 January and end on Saturday 30&nbsp;January.</Paragraph>
       </TextContainer>
 
       <KnockoutRounds knockoutRoundData={gameData.knockout} />
+
+      <TextContainer>
```

Comment:
```suggestion
      {/* NOTE: Move active round to the top when they start, but only once they start (so that the current round is always at the top */}
      <TextContainer>
```